packages/api-plugin-fulfillment-method-shipping-flat-rate/src/mutations/createFlatRateFulfillmentMethod.js

```diff
@@ -0,0 +1,53 @@
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import ReactionError from "@reactioncommerce/reaction-error";
+import methodSchema from "../util/methodSchema.js";
+
+const inputSchema = new SimpleSchema({
+  method: methodSchema,
+  shopId: String
+});
+
+/**
+ * @method createFlatRateFulfillmentMethodMutation
+ * @summary Creates a flat rate fulfillment method
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - Input (see SimpleSchema)
+ * @returns {Promise<Object>} An object with a `method` property containing the created method
+ */
+export default async function createFlatRateFulfillmentMethodMutation(context, input) {
+  const cleanedInput = inputSchema.clean(input); // add default values and such
+  inputSchema.validate(cleanedInput);
+
+  const { method: inputMethod, shopId } = cleanedInput;
+  const { collections } = context;
```

Comment:
These destructuring statements should be combined

Reply:
Updated